stm/vernac_classifier.ml

```diff
@@ -199,11 +199,8 @@ let classify_vernac e =
   in
   let rec static_control_classifier ~poly = function
     | VernacExpr (f, e) ->
-      let poly = List.fold_left (fun poly f ->
-          match f with
-          | VernacPolymorphic b -> b
-          | (VernacProgram | VernacLocal _) -> poly
-        ) poly f in
+      let _, atts = Vernacentries.attributes_of_flags f Vernacinterp.{ loc = None ; locality = None ; polymorphic = poly ; program = false ; coercion = false; } in
```

Comment:
I don't know if it exists (if not it should be added), but I'd like to see a `empty_flags` or `default_flags` here rather than the this record, otherwise every time you add a field or change one you touch the stm for no reason.